docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,44 @@
+
+<img alt="Liana time locked coins" src="../assets/liana-coins.png" />
+
+But this reconciliation comes at a cost to privacy. These transactions create an identifiable on-chain footprint, making it easier for on-chain surveillance to estimate a user's balance and transaction history. This is primarily due to the [common input ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), which assumes all inputs in a transaction belong to the same entity. Combined with unique transaction patterns that act like fingerprints, these heuristics allow on-chain analysts to cluster a user's addresses and glean sensitive information.
+
+## Payjoin as a Privacy-Preserving Refresh
+
+This is where Payjoin shines. It allows Liana users to refresh near-expiry coins during normal payments. For example, when a user receives a payment via Payjoin, they can opportunistically include expiring coins to refresh their timelocks. This same logic can apply to self-spends—users can initiate a Payjoin when sending a transaction, embedding refresh into routine activity.
+
+Whether as sender or receiver, users benefit from lower fees, improved privacy, and preservation of their inheritance policies.
+
+<img alt="Liana Payjoin" src="../assets/liana-batch-payment.svg" />
+
+## Caveats
+
+It’s worth noting that the common-input heuristic is not defeated when using Payjoin with P2WSH inputs—unless both participants share identical or structurally similar descriptors. This is because the spending script is revealed on-chain, allowing fingerprinting based on script templates.
+
+However, Taproot inputs do not suffer from this issue. With Taproot’s key-path spending, the recovery script remains hidden unless explicitly revealed. This makes it possible to conduct Payjoin transactions between users with different wallet policies while breaking the common-input heuristic.
+
+## Proof of Concept and Next Steps
+
+During the MIT Bitcoin Hackathon, the Payjoin team developed a working proof of concept for both the Payjoin receiver and sender integrated with Liana. You can view the prototype [here](https://github.com/0xBEEFCAF3/liana/tree/payjoin-hackathon). In just 30 hours, the team successfully implemented the core functionality.
+
+Looking ahead, we’re coordinating closely with the Liana engineering team to bring this integration to production. This involves rigorous testing, refining the transaction construction logic, and ensuring compatibility with Liana’s policy enforcement mechanisms. Together, we’ll work toward a robust, production-ready release that empowers users with a privacy-preserving, fee-efficient way to manage their recovery paths.
+
+This integration brings privacy and practicality to Bitcoin inheritance, showing how simple batching protocols like Payjoin can improve user experience without compromising privacy.
```

Comment:
A lot of focus on the privacy aspects here, which is fine in this article and probably the major use-case for a Liana-type integration, but just noting that our emphasis if we care about adoption should be fee-and-efficiency savings.

Reply:
What you have is good I think, I guess I figured this struck me as privacy-centric while reading it, though our goal in _general_ with payjoin is to move the marketing from "privacy is primary goal with time/fee savings as byproduct" to vice-versa. But in the Liana case I think what you have is good 👍